core/module.go

```diff
@@ -23,7 +23,7 @@ type Module struct {
 	NameField string `field:"true" name:"name" doc:"The name of the module"`
 
 	// The doc string of the module, if any
-	Description *string `field:"true" doc:"The doc string of the module, if any"`
+	Description string `field:"true" doc:"The doc string of the module, if any"`
```

Comment:
This is a tiny API change where now it's non-null and will just default to an empty string. I like that more anyway, just making a note since it wasn't mentioned in the description.

Reply:
Yeah, I did that for consistency with `Function.description` and `ObjectTypeDef.description`.